components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationConstants.java

```diff
@@ -132,6 +132,8 @@ public static class ApplicationTableColumns {
         public static final String IMAGE_URL = "IMAGE_URL";
         public static final String ACCESS_URL = "ACCESS_URL";
 
+        public static final String INBOUND_AUTH_KEY = "INBOUND_AUTH_KEY";
```

Comment:
Shall we remove line no 135?

Reply:
As we use it in applicationDAOImpl, added it to a new class named as ApplicationInboundTableColumns.